packages/block-library/src/cover/test/edit.js

```diff
@@ -371,13 +371,12 @@ describe( 'Cover block', () => {
 						name: 'Styles',
 					} )
 				);
-				await userEvent.clear(
-					screen.getByLabelText( 'Minimum height of cover' )
-				);
-				await userEvent.type(
-					screen.getByLabelText( 'Minimum height of cover' ),
-					'300'
-				);
+
+				const [ heightControl ] =
+					screen.getAllByLabelText( /Minimum height/ );
```

Comment:
```suggestion
				const heightControl = screen.getByRole( 'spinbutton', {
					name: 'Minimum height',
				} );
```

It is better to [use accessible selectors](https://github.com/WordPress/gutenberg/blob/fix/issue-56020/docs/contributors/code/e2e/README.md#use-accessible-selectors).